<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddCourse</class>
 <widget class="QDialog" name="DialogAddCourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить курс</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>10</y>
     <width>351</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Коэфф. за семинары</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="practCoefSpinBox"/>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="testCoefSpinBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Коэфф. за практические</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="semCoeffSpinBox"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="courseNameLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Коэфф. за домашние</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Название курса:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="hwCoefSpinBox"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Коэфф. за экзамен</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="examCoefSpinBox"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Коэфф. за тесты</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="addCourseButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить курс</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
